libs/wai-utilities/src/Network/Wai/Utilities/Server.hs

```diff
@@ -264,12 +265,19 @@ emitLByteString lbs = do
 --------------------------------------------------------------------------------
 -- Utilities
 
+-- | 'onError' and 'catchErrors' support both the metrics-core ('Right') and the prometheus
+-- package introduced for spar ('Left').
+type OnErrorMetrics = Either Prm.Counter Metrics
```

Comment:
Until we get transitioned over this should actually use both right? It should log all errors to both monitoring services.

Reply:
yes, so something like `type OnErrorMetrics = [Either Prm.Counter Metrics]`.